scripts/microbe_stage/microbe_operations.as

```diff
@@ -852,19 +852,6 @@ ObjectID spawnMicrobe(CellStageWorld@ world, Float3 pos, const string &in specie
     auto speciesEntity = findSpeciesEntityByName(world, speciesName);
     auto species = world.GetComponent_SpeciesComponent(speciesEntity);
 
-    // TODO: Why is this here with the separate spawnBacteria function existing?
-    // Bacteria get scaled to half size
-    if(species.isBacteria){
```

Comment:
Unless you did something to make this work without this, this is very much needed.
Also you need to start the new branch from master to have a clean second pull request.

Reply:
ok